experiments/probabilistic_domain_adaptation/livecell/common.py

```diff
@@ -212,11 +215,11 @@ def _parse_aug(aug):
     return loader
 
 
-def get_supervised_loader(args, split, cell_type):
-    patch_shape = (512, 512)
```

Comment:
Let's change this in the unsupervised loader as well.

Reply:
Updated this as well (however, I am really curious to see how the joint training setup works with more context - (512, 512))